src/reducers/common.ts

```diff
@@ -38,7 +39,8 @@ export const commonStateReducer = (
         };
       } else {
         const currData = get(state.data, action.path);
-        const newState = set(state.data, action.path, action.updater(currData));
+        const data = action.updater(currData);
```

Comment:
shouldn't the variables be named:
`currData` -> `oldData`
`data` -> `newData`
?